src/converters.js

```diff
@@ -622,37 +625,37 @@ export function modelChangePostFixer( document ) {
 // Helper function for post fixer callback. Performs fixing of model `listElement` items indent attribute. Checks the model at the
 // `changePosition`. Looks at the node before position where change occurred and uses that node as a reference for following list items.
 function _fixItemsIndent( changePosition, document, batch ) {
-	const prevItem = changePosition.nodeBefore;
 	let nextItem = changePosition.nodeAfter;
 
 	if ( nextItem && nextItem.name == 'listItem' ) {
-		// This is the maximum indent that following model list item may have.
-		const maxIndent = prevItem && prevItem.is( 'listItem' ) ? prevItem.getAttribute( 'indent' ) + 1 : 0;
-
-		// Check how much the next item needs to be outdented.
-		let outdentBy = nextItem.getAttribute( 'indent' ) - maxIndent;
-		const items = [];
-
-		while ( nextItem && nextItem.name == 'listItem' && nextItem.getAttribute( 'indent' ) > maxIndent ) {
-			if ( outdentBy > nextItem.getAttribute( 'indent' ) ) {
-				outdentBy = nextItem.getAttribute( 'indent' );
-			}
+		document.enqueueChanges( () => {
```

Comment:
Do I presume correctly that this was just a cosmetic change here? If so, please make such changes in separate commits because it's then hard to check what has really changed.